File: vehiculo.py
class Vehiculo:
    def __init__(self, marca, modelo, anio):
        self._marca = marca
        self._modelo = modelo
        self._anio = anio

    def get_marca(self):
        return self._marca

    def set_marca(self, marca):
        self._marca = marca

# Ejemplo de uso
vehiculo1 = Vehiculo("Toyota", "Corolla", 2020)
print(vehiculo1.get_marca())
vehiculo1.set_marca("Honda")
print(vehiculo1.get_marca())

File: vehiculo.php
<?php
class Vehiculo {
    // Propiedades
    private $marca;
    private $modelo;
    private $anio;

    // Constructor
    public function __construct($marca, $modelo, $anio) {
        $this->marca = $marca;
        $this->modelo = $modelo;
        $this->anio = $anio;
    }

    // Método para obtener la marca
    public function getMarca() {
        return $this->marca;
    }

    // Método para establecer la marca
    public function setMarca($marca) {
        $this->marca = $marca;
    }
}

// Ejemplo de uso
$vehiculo1 = new Vehiculo("Toyota", "Corolla", 2020);
echo $vehiculo1->getMarca();
$vehiculo1->setMarca("Honda");
echo $vehiculo1->getMarca();
?>
